src/patternfly.module.js

```diff
@@ -7,9 +7,10 @@
 angular.module('patternfly', [
   'patternfly.autofocus',
   'patternfly.card',
+  'patternfly.charts',
```

Comment:
Maybe update the charts.module.js comment saying that they must inject patternfly.charts and also duplicate in the ngdocs for the c3 directive?